CIP-0001/README.md

```diff
@@ -140,6 +141,28 @@ CIP-0010
 
 ```
 
+##### Translations
+
+While CIPs are mainly technical documents meant to be read primarily by developers -- and thus often written in English; some may be translated into various languages to increase their outreach. Any file in a CIP folder may also include translated content satisfying the following rules:
+
+- Any translated file shall share a common basename with its original source.
+
+- Translation file basenames must have a suffix using an [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) language code, separated by a dot `.` character. (e.g. `README.fr.md`).
+
+- When no language code is provided as suffix, the default language for the document is assumed to be English (UK/US).
+
+- Translated CIPs (i.e. `README` files), must not include the original preamble. They must, however, include the following preamble as yaml frontmatter data:
+
+Field          | Description
+---            | ---
+`Source`       | A link to the original CIP document
```

Comment:
```suggestion
`Source`       | The canonical GitHub link to the original CIP document
```
I believe we need to say "canonical" because links to `README.md` files are often supplied by giving the containing folder (as often done for CIPs themselves).  Only the canonical `README.md` link will have a corresponding list of commit hashes for the `Revision` field.

Likewise (as I mentioned at the meeting), if we combine `Source` and `Revision` into a single link to the commit hash, then the more naive GitHub visitors might not be able to follow the UI to the original document.